modules/tinymce/changelog.txt

```diff
@@ -1,5 +1,5 @@
 Version 5.5.0 (TBD)
-
+    Fixed `media` not saving alternative source url in some situations.
```

Comment:
There's a few minor grammar issues here, there should be no fullstop at the end and the JIRA number is missing. So it should probably be something like this:
```
Fixed the `media` plugin not saving the alternative source url in some situations #TINY-4113
```

Reply:
Swing and a miss. Better luck next time.